stable/openvpn/templates/certs-pvc.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.persistence.Enabled }}
+
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: {{ .Values.persistence.StorageClass | quote }}
```

Comment:
Should beta storage be the default since it is more stable?  It will will default to alpha per your example.